src/components/SharedComponents/ButtonBar.tsx

```diff
@@ -51,7 +51,7 @@ const ButtonBar = ( {
         const {
           title, onPress, isPrimary, ...props
         } = button;
-        console.log( "props", { ...props } );
+
         return (
           <Button
```

Comment:
We should add a `key` to this component since we're mapping over an array and currently getting a warning about that. (Not a new problem, but let's fix it).